packages/xo-web/src/xo-app/vm/vusb-create-modal.js

```diff
@@ -0,0 +1,27 @@
+import _ from 'intl'
+import Component from 'base-component'
+import React from 'react'
+import { Select } from 'form'
+
+const USB_GROUP_RENDERER = pusb => `${pusb.description} (${pusb.version})`
```

Comment:
Why "GROUP"?

Reply:
My bad. Previously it was used to display `USB_group`.